File: classexercise/java/Palindrome.java
import java.util.Scanner;
public class Palindrome{
	public static void main(String []args){
	Scanner input = new Scanner(System.in);

	System.out.print("Input a five-digit number: ");
	int number = input.nextInt();
 	
	int digit1 = number / 10000 % 10;
	int digit2 = number / 1000 % 10;
	int digit3 = number / 100 % 10;
	int digit4 = number / 10 % 10;
	int digit5 = number / 1 % 10;

	if(digit1 == digit5 && digit2 == digit4){
	System.out.print(number);
	System.out.print(" is a palindrome.");
	}else{
	System.out.print(number);
	System.out.print(" is not a palindrome.");
	}





}
}

File: assignments/chapter3/palindrome.py
number = int(input('Enter a five digit number: '))
digit1 = int(number / 10000 % 10)
digit2 = int(number / 1000 % 10)
digit3 = int(number / 100 % 10)
digit4 = int(number / 10 % 10)
digit5 = int(number / 1 % 10)

if(digit1 == digit5 and digit2 == digit4):
	print('THIS NUMBER IS A PALINDROME')
else: 
	print('THIS NUMBER IS NOT A PALINDROME')



